.github/actions/tests/local-environment-tests/action.yml

```diff
@@ -42,27 +42,12 @@ runs:
       with:
         name: partner-chains-node-artifact
         path: dev/local-environment/configurations/pc-contracts-cli/overrides/
-    - name: checkout pc-tests ref_name branch
-      id: pc-tests
-      uses: actions/checkout@v4
-      continue-on-error: true
-      with:
-        repository: input-output-hk/sidechains-tests
-        token: ${{ env.ACTIONS_PAT }}
-        ref: ${{ github.head_ref || github.ref_name }}
-        path: dev/local-environment/configurations/tests/e2e-tests
-    - name: checkout pc-tests master (fallback)
-      if: steps.pc-tests.outcome == 'failure'
-      uses: actions/checkout@v4
-      with:
-        repository: input-output-hk/sidechains-tests
-        token: ${{ env.ACTIONS_PAT }}
-        ref: master
-        path: dev/local-environment/configurations/tests/e2e-tests
```

Comment:
@ladamesny have you checked pc-demo? I can't see any changes in `dev/local-environment/configurations/tests/entrypoint.sh` so it may have trouble finding the tests

Reply:
apologies, I see the new lines :)

Reply:
yeah I didn't have to change it, because that file still references `./devnet/local-environment/configurations/tests/e2e-tests` folder, I updated the script to copy over our test files from `./E2E-tests/` to `./devnet/local-environment/configurations/tests/e2e-tests`.